trainings/sourcegraph-101/talk-track.md

```diff
@@ -0,0 +1,327 @@
+
+You can also enable the Notepad feature by clicking "enable Notepad" up here in the top right of the screen.
+
+*Show that.*
+
+That will allow you to create a new note snippet from your searches. So you can see that if I search for `context:global new auth provider repo:sourcegraph patternType:regexp`, it prompts me to add that search to my notepad. I can then convert the notepad to a notebook.
+
+I want to pause here—do folks have questions about creating notebooks? Can anyone share a situation where they might use the notebooks feature?
+
+### Conclusion
+
+With notebooks, you can share code context and explanations with your teammates. The Notepad feature lets you build notebooks on the go.
+
+### Resources 
+
+* [Notebooks documentation](https://docs.sourcegraph.com/notebooks)
+
+## Unit 6: Extensions
+
+**Learning goals:** The user will understand how our extensions platform functions and that IDE extensions exist.
+
+### Intro to the extensions page
+
+The final thing that I want to touch on is our extensions platform. You can access that by clicking the puzzle icon at the top right here.
+
+*Navigate to https://demo.sourcegraph.com/extensions*
+
+### Editor extensions 
+
+We integrate with a variety of services here, but the one I particularly want to draw your attention to is this "code editors" section. If you click into that, you'll see a list of supported editors and a generic "open in editor" extension. 
+
+*Navigate to https://demo.sourcegraph.com/extensions?category=Code+editors*
+
+You'll want to enable the appropriate extension for your editor, and then click on the extension name to view the customization settings you'll need to configure. 
+
+*Go to https://demo.sourcegraph.com/extensions/sourcegraph/open-in-editor and highlight the different config options on that page.*
+
+This does rely on having the code locally on your machine—but if you do, you'll have an "open in editor" button in Sourcegraph to open the file locally. And if you want to navigate from your editor back into Sourcegraph, you can do so with the extensions here.
+
+*Open https://docs.sourcegraph.com/integration/editor*
+
+We'll share this link with you after the presentation so you can install the right extension for you. You'll just need to point it at the right Sourcegraph instance in your settings. 
+
+### VSCode extension
+
+For VSCode users—and soon JetBrains users!—we have a more robust integration. With VSCode, you actually can do everythign we've seen so far directly in VSCode.
+
+*Share VSCode, open the Sourcegraph extension, run a search*
+
+So as you can see here, I'm able to search all of the same code that's on my Sourcegraph instance, no matter what's local. We really encourage VSCode users to install this extension, since it gives you all of Sourcegraph's search power right in the environment you're already using.
```

Comment:
```suggestion
So as you can see here, I'm able to search all of the same code that's on my Sourcegraph instance, no matter what's local. This means without cloning every repository to search everywhere in VSCode, Sourcegraph can search all local AND remote repositories that you have access to on your code host. We really encourage VSCode users to install this extension, since it gives you all of Sourcegraph's search power right in the environment you're already using.
```

I think I called this out before in person, but this is a hugely powerful thing that I don't think is conveyed well in our current scripting, so I tried to wordsmith it a bit. Open to thoughts 👍

Reply:
I like this, thanks!